### PR TITLE
Ensure GetElement are marked as having an immediate operand for Arm64

### PR DESCRIPTION
**Pull request by tannergooding**

### CODE REVIEW

**Comment by tannergooding** [comment]
CC. @EgorBo, this fixes the minor codegen issue we found

**Comment by tannergooding** [comment]
Guess it's not so minor, since we weren't doing containment at all (for the address or for the index):
```
Overall (-34,888 bytes)
MinOpts (-5,128 bytes)
FullOpts (-29,760 bytes)
```

Most of these are simple cases such as:
```diff
-            mov     w0, #2
             dup     s16, v16.s[2]
```
-or-
```diff
-            mov     w6, #1
             umov    x6, v16.d[1]
```

But then we also get cases such as:
```diff
-            ldr     q17, [fp, #0x30]	// [V02 loc0]
-            mov     w0, #3
-            dup     s17, v17.s[3]
+            ldr     s17, [fp, #0x3C]	// [V02 loc0+0x0c]
```

There's a lot of code that ends up extracting or moving elements, so this gets some decent payoffs overall. It does, however, look like there's some other missed containment opportunity as well:
```diff
-            mov     w1, #2
             mov     v8.d[1], v9.d[0]
             dup     s0, v8.s[2]
+            mov     w1, #2
             ldr     x0, [x20, #0x08]
```

Going to see if I can identify where that is, but I wouldn't block this on that.

File: src/coreclr/jit/hwintrinsiclistarm64.h [hunk 1]
```diff
@@ -155,7 +155,7 @@ HARDWARE_INTRINSIC(Vector128,     EqualsAny,
...
-HARDWARE_INTRINSIC(Vector128,     GetElement,                                                       16,      2,      true,  {INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_umov,           INS_umov,           INS_dup,            INS_dup},         HW_Category_Helper,                HW_Flag_SpecialImport|HW_Flag_SpecialCodeGen|HW_Flag_BaseTypeFromFirstArg)
+HARDWARE_INTRINSIC(Vector128,     GetElement,                                                       16,      2,      true,  {INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_umov,           INS_umov,           INS_dup,            INS_dup},         HW_Category_Helper,                HW_Flag_SpecialImport|HW_Flag_SpecialCodeGen|HW_Flag_BaseTypeFromFirstArg|HW_Flag_SupportsContainment)
```
Comment (EgorBo) [inline src/coreclr/jit/hwintrinsiclistarm64.h:158]: one thing I noticed that GetElement is not marked as `HW_Flag_SupportsContainment` for x64 but it still works there 🤔

**Review by EgorBo — APPROVED** [review]

File: src/coreclr/jit/hwintrinsiclistarm64.h (hunk 1 above)
Comment (tannergooding) [reply src/coreclr/jit/hwintrinsiclistarm64.h:158]: They’re functionally inverses of each other.

On Arm64 containment is a rare consideration and so is opt-in, while on x64 it’s a common thing for almost every operation and is instead opt-out

**Pull request merged**